### ISSUE TITLE
Wish: store apps on sdcard inteal/external

### ISSUE THREAD

**Issue by Spielmops** [kitsunyan/foxy-droid#7]
Until now I had to use the fdroid-app 0.102 (instead of the crazy new version), which stores all downloaded apks in it's cache. This cache can get very big after some weeks. Why not do it like Aurora and store all apks in a folder on storage. Additional advantage: it would be easier to share apk with an USB-stick.

Spielmops

**Comment by kitsunyan** [comment]
On most Android phones /data and /sdcard share the same partition (/sdcard is an emulated vfat-compatible device). In contrast, external SD card requires using SAF to access data.

This F-Droid have very aggressive cache cleaner, it removes APKs after day or two (depends on whether they fully downloaded or just partially). I'd preferred to not add storage access permission at all and not touch external storage at all. Especially I'd like to not use SAF, it's extremely disgusting technology.

**Comment by 5c0tt-b0t** [comment]
Agreed that the Storage Access Framework isnt great! and its slow! but storing the APK on device (internal) doesnt need this, just storage permission.

**Comment by kitsunyan** [comment]
Actually, it doesn't even need storage permission to store in `Android/data/package.name`, but I still think that it's redundant since /data and internal storage are on the same partition in most cases and cache cleaner is very aggressive.

**Comment by Spielmops** [comment]
The difference between /data and internal storage: I can easyly copy downladed apps onto my USB-stick and install them on other devices. Or share them via bluetooth or nextcloud ...
